offload/cmake/OpenMPTesting.cmake

```diff
@@ -136,7 +125,7 @@ if (${OPENMP_STANDALONE_BUILD})
   # project is built which is too late for detecting the compiler...
   file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/DetectTestCompiler)
   execute_process(
-    COMMAND ${CMAKE_COMMAND} -G${CMAKE_GENERATOR} ${CMAKE_CURRENT_LIST_DIR}/DetectTestCompiler
+    COMMAND ${CMAKE_COMMAND} -G${CMAKE_GENERATOR} ${CMAKE_SOURCE_DIR}/../openmp/cmake/DetectTestCompiler
```

Comment:
It's nearly always wrong to use CMAKE_SOURCE_DIR instead of CMAKE_CURRENT_SOURCE_DIR

Reply:
Fixed.